packages/expo-constants/src/Constants.ts

```diff
@@ -106,11 +110,46 @@ const constants = {
     }
     return nativeConstants.linkingUri;
   },
+  get manifest() {
+    return getManifest();
+  },
+  get manifestDerivedMethods() {
+    if ('updateMetadata' in this.manifest) {
```

Comment:
```suggestion
    if ('metadata' in this.manifest) {
```

Reply:
I'll do this all in one follow-up commit since there are a lot of other places that assume `updateMetadata`.